pytorch_lightning/trainer/trainer.py

```diff
@@ -883,6 +883,18 @@ def progress_bar_dict(self) -> dict:
         ref_model = self.model if not self.data_parallel else self.model.module
         return dict(**ref_model.get_progress_bar_dict(), **self.progress_bar_metrics)
 
+    @property
+    def disable_validation(self) -> bool:
+        """ Check if validation is disabled during training. """
+        disable_validation = not (self.is_overridden('validation_step') and self.limit_val_batches > 0) \
```

Comment:
when all are not, would it be easier to write `enable_validation` and negate it here? (vise verso to this state)

Reply:
done